docs/administration/internal-grid/grid-rendering-customization.md

```diff
@@ -3,9 +3,36 @@
 Default [Twig](https://twig.symfony.com/) template for rendering each grid can be found in [`src/Resources/views/Admin/Grid/Grid.html.twig`](https://github.com/shopsys/shopsys/blob/master/packages/framework/src/Resources/views/Admin/Grid/Grid.html.twig).
 The template is composed of a set of Twig blocks, and you can override any of them when there is a need for customization of the default appearance.
 
-To customize your grid, you need to create a new template extending the original one, override appropriate blocks, and then set the template as a theme of your grid using `Grid::setTheme` method.
+To customize your grid, you can use on of the following approaches:
 
-## Blocks that are being overridden at most
+## 1. Define your own column template
+
+You can create your own template and set is as a parameter to the column definition. 
+
+Create a new template in your project, e.g. `templates/Admin/Grid/my_awesome_column.html.twig`:
+```
+<span class="my-awesome-column">
+    {{ value }}
+</span>
+```
+
+In template, you can use following variables:
+- `value` - the value of the column
+- `row` - the whole row of the grid. You can access to the row data using the column name as a key, e.g. `row['columnName']` or as a property, e.g. `row.columnName`
+- `column` - the column object itself
```

Comment:
it would be nice to add "(i.e. instance of XX class)"

Reply:
fixed